modules/installation-azure-permissions.adoc

```diff
@@ -6,10 +6,10 @@
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
 
-Your Microsoft Azure account must have the following roles for the subscription
-that you use:
+{product-title} needs a service principal so it can manage Microsoft Azure resources. Before you can create a service principal, your Azure account subscription must have the following roles:
```

Comment:
@patrickdillon We've change the wording for clarity. Is this still okay?

Reply:
Yes LGTM.